### ISSUE TITLE
Mod Tweaks/Removals

### ISSUE THREAD

**Issue by IppSwitch** [kreezxil/World-of-Dragons#287]
Issues with; 
(furniture-6.3.0-1.12.2.jar)
MrCrayFish Funature Mod Printer
(Apotheosis-1.12.2-1.10.2.jar)
Apotheosis - Knowledge of ages.
Apotheosis - Hell Shelves.
(rats-1.2.0.jar)
Rats
(SlashBladeTic-V1.0.9.1-MC1.12.2.jar)
Slashblades

The Printer from MrCrayFish can be created cheap (2 blocks of iron, 1 piece of redstone, 1 piece of paper, 5 pieces of stone) and can then be used to make cheap copies of Enchantment books. 
(This is heresay from what i have been told.)

Pending request until someone can validate how easy it is to make these books.

Apotheosis - Knowledge of ages
Using this on it's turns all loot a mob would generate and turns it into xp.
Mixing this with Looting (any level) turns this into a MASSIVE xp gain. 
In a space of a day a player gained 643 levels. Total play time was 38hrs for the player in question.

Request removal

Apotheosis - Hell Shelves.
These shelves give enchantments that are not in the vanilla game and also boost vanilla enchanting to overkill. (Up to level 17)  This bow was created with these added bonuses and was dealing at least 500-1000dmg over 30 seconds. 
![image](https://user-images.githubusercontent.com/38470419/61625928-83fa0b80-acae-11e9-9568-2e2f45d36a48.png)

Request Tweak or Removal

Rats - Confiti Biyaldi
Using this food renders any player capable of both surviving and soloing a T3 dragon.
(Was T3 ice dragon that was tested)

Request Tweak or Removal.
 
Slashblades. 
They have further range than anything (Not including ranged attacks)
They can deflect all projectiles (Including a dragon ball attack)
They can stunlock all Non-flying mobs. (haven't tested on dragons)

Request Tweak or Removal.

Tested various armors and weapons enchanted/upgraded by Apotheosis methods.
Players became incredibly powerful. Capable of surviving 4-5 direct dragon hits. 
If players mix Confiti into the mix, the player is 100% capable of surviving and killing a T3 dragon. I did not test T4 or T5

Edit*: Added Tweak to Rats.

**Comment by IppSwitch** [comment]
Vanilla Nether chests.
90% of chests i go threw in the nether have a Advanced Machine frame in them. 
They are not easy to make. But with this you don't Need to make them.

Request Tweak or removal.

**Comment by Yodown814** [comment]
Yes. I have changed my mind since I heard someone saying hell shelves are going to be removed, and I personally have been working on enchanting for days now, I like them, so I strongly disagree and against this tweaks/removal. If you guys think that they are **overpowered**, buff the monsters. You should not be removing something that is the core style to a mod, enchanting is all Apotheosis about. First of all, the bow above is from one of my teammates, and I have a similar one, so I know exactly how to get it and what it does. Trust me, it's **NOT** so easy that we made it this far. From what you're saying, Knowledge of the Ages is basically a busted enchantment that gives the player a ton of XP, but did you know, that it is **NOT** obtainable through regular enchantment setup(which is 16 bookshelves around an enchantment table). It requires the player to have been to the nether and found the nether castle, and also have killed ghast and started brewing. I don't think this is that easy for a new player to achieve? Second of all, have all 16 hell shelves isn't even gonna put you to the maximum, you literally still have to enchant the hell shelves all by 150 level/1 to get to the maximum enchanting level, all that is easy? Another thing I don't get is, you guys want a full gear player **still** unable to even solo a t3 dragon or what? I understand "dragons aren't supposed to be soloed", but I don't get why is that necessary to just nerfed that many things at a time. In the opposite though, I will rather you guys make it that t5 dragons are invincible by 1 player in any circumstances, but at least make the t3 dragons defeatable by 1 full gear player. Also for rats(or confit byaldi to be exact), it is **NOT**, again, so easy to obtain. You need a total of 8 different types of potions, and also golds to be able to craft just 1 enchanted golden apple(which requires 6 to make the chef upgrade), and in order to brew those potions, the player needs to go to the Nether and the End to be able to get all the materials they need. So for that, I don't think confit blyadi is going to spread and cheap for every else to make and use. Lastly, for SlashBlade. A mod that I recommended, of course I won't like it to be removed or tweaked. Targeted by your beliefs, they **have a long attack range**. I mean, what do they look like? a dagger? no, a katana. I don't know if you have seen any IRL, but they are long. They **deflect all projectiles**. Welp isn't that **EXACTLY** what a sword does? deflects a fireball from a ghast(I don't know what exactly and how exactly it deflects a fireball from dragon when you literally said you **haven't tested** on t5 dragons in other statements). They **stunlock all Non-flying mobs**. I'm sorry, but do dragons **fly** or **no**?

**Comment by Yodown814** [comment]
This is the **last** and **only** comment that I will be commenting, no further discussion over this. You keep it you keep it, you get rid of it you get rid of it.

**Comment by iamweirdie** [comment]
>I like [hell shelves], so I strongly disagree and against this tweaks/removal. If you guys think that they are **overpowered**, buff the monsters. You should not be removing something that is the core style to a mod, enchanting is all Apotheosis about.

This core comment right here might be the reason Apotheosis gets removed- because the mod IS overpowered. buff monsters is NOT the solution, as new players are still dying to the present dangers on the battlefield. Buff monsters will only make it harder and harder for new players, discouraging them to play.

![image](https://user-images.githubusercontent.com/34356883/61655054-9d7d6200-ac72-11e9-889c-8be89abfd620.png) This graph here is pure fairness, as you get better and better on the x axis, you can face better stuff on the y axis. very simple

![image](https://user-images.githubusercontent.com/34356883/61655212-f2b97380-ac72-11e9-8a4d-fef8c400f8fd.png) In this instance, players face better stuff, but at MUCH crazier speeds. notice how new players at 0 still ARE 0. This is what happens when there's an exploit or overpowered mod. In this situation, senior players will reign 100%, and we admins want to level that playing field so new people have a better chance.

![image](https://user-images.githubusercontent.com/38470419/61625928-83fa0b80-acae-11e9-9568-2e2f45d36a48.png)
>  the bow above is from one of my teammates, and I have a similar one, so I know exactly how to get it and what it does. Trust me, it's **NOT** so easy that we made it this far.

This says nothing, NOTHING, about your process of how to get the bow. In this statement, we, the reader, need to be convinced WHY it wasn't easy to obtain the bow. What did you get? What did you smelt? How long did smelting take? Was it a lot of sitting time? Could I read Romance of the Three Kingdoms while I wait to get this item? I'm sorry, but this is not a good enough argument.

>From what you're saying, Knowledge of the Ages is basically a busted enchantment that gives the player a ton of XP, but did you know, that it is **NOT** obtainable through regular enchantment setup(which is 16 bookshelves around an enchantment table). It requires the player to have been to the nether and found the nether castle, and also have killed ghast and started brewing. I don't think this is that easy for a new player to achieve?

Therein lies the issue: the ghast actually IS an easy mob to find and kill. 1 lava lake and some iron is all a new player really needs to get a nether portal up and running. Finding a nether fortress is difficult on its own, but I guarantee you, people are willing to share their nether fortresses, and that can't be stopped. We would need to configure nether fortresses to be rarer if we didn't modify recipes, and that ALSO means regenerating the nether.

>Second of all, have all 16 hell shelves isn't even gonna put you to the maximum, you literally still have to enchant the hell shelves all by 150 level/1 to get to the maximum enchanting level, all that is easy?

Yes. It takes only 1 person to share their own Knowledge of Ages system to really kickstart that particular curve. New players would be more easily exposed to higher end weapons because some old player just hands them out, which we don't like. If an old player hands a new player an old weapon, we still expect the new player to have some significant strength or learning curve. For other items such as a powerful bow, they take work and are not so easy to hand out. For enchantments paired with apotheosis, they're too easy to hand out.

>Another thing I don't get is, you guys want a full gear player **still** unable to even solo a t3 dragon or what? I understand "dragons aren't supposed to be soloed", but I don't get why is that necessary to just nerfed that many things at a time.

MOST, if not EVERY, dragon, should not be solo'd. We nerfed/removed things because it's much easier than buffing the dragon and destroying new players harder. As more overpowered mods get introduced, the line becomes more and more steep. We would like to let new players play the game without relying on everyone to barf free stuff on them.

>In the opposite though, I will rather you guys make it that t5 dragons are invincible by 1 player in any circumstances, but at least make the t3 dragons defeatable by 1 full gear player.

Ice and Fire has its own curve that we agree to and abide by. a t1 dragon =/= a baby lizard, it is a difficult mob AFTER many other Ice and Fire mobs. We have yet to introduce THOSE mobs into the equation. If a t3 dragon can be defeated by a solo player, then it makes the rest of those mobs- the gorgon, the cyclops, the troll- much easier. We cannot increase the incline of just 1 or 2 dragons, the incline applies to ALL dragons.

>Also for rats(or confit byaldi to be exact), it is **NOT**, again, so easy to obtain. You need a total of 8 different types of potions, and also golds to be able to craft just 1 enchanted golden apple(which requires 6 to make the chef upgrade), and in order to brew those potions, the player needs to go to the Nether and the End to be able to get all the materials they need. So for that, I don't think confit blyadi is going to spread and cheap for every else to make and use. 

The issue with the confit is not that it's easy to obtain, the issue is it boosts powerful items to the stage of OVERpowered. That's the reason we need to take it out. To counter the specific point I stated, you would need to find a strong enough weapon, pair it with the confit, and tell me that the weapon is not overpowered. This is classified as an exploit.

>Lastly, for SlashBlade. A mod that I recommended, of course I won't like it to be removed or tweaked. Targeted by your beliefs, they **have a long attack range**. I mean, what do they look like? a dagger? no, a katana. I don't know if you have seen any IRL, but they are long.

![image](https://images-na.ssl-images-amazon.com/images/I/51heTiecZmL._SY550_.jpg)
this is a katana held by a scale human being.
![image](https://res.cloudinary.com/teepublic/image/private/s--mThMXK1S--/t_Preview/b_rgb:ffffff,c_limit,f_jpg,h_630,q_90,w_630/v1547347337/production/designs/3959621_0.jpg)
this is how we see slashblade.

>They **deflect all projectiles**. Welp isn't that **EXACTLY** what a sword does? deflects a fireball from a ghast(I don't know what exactly and how exactly it deflects a fireball from dragon when you literally said you **haven't tested** on t5 dragons in other statements). They **stunlock all Non-flying mobs**. I'm sorry, but do dragons **fly** or **no**?

No, swords apparently do not deflect all projectiles, not according to the other swords that we have in this modpack.

>This is the last and only comment that I will be commenting, no further discussion over this. You keep it you keep it, you get rid of it you get rid of it.

YESTERDAY, at 3:20 PST, on wod-18, you said, and i quote,
 **ima not going to make any more disussions or suggestions on githubs anymore**
so i highly doubt this is the last thing you'll say on github.

For delivering a point, this short essay you've written has holes that can easily be poked. I understand that this is not an academic essay, but they're also easy rules to strengthen arguments in general.

>As a rule, if you raise a question in an academic essay, you will be expected either to provide a satisfactory answer to that question, or to address why it cannot be effectively resolved.

>If the question comes across as idle speculation or as a leading rhetorical device, this may lower the tone of the essay. Be sure that the question actually contributes something useful to the argument you are building.
- [source](https://www.quora.com/Can-you-use-questions-in-an-academic-essay)

Some of your comments were loaded with pure attitude, and while you may actually be packing attitude at the time of writing, your comments should not reflect such, as an emotional argument is easier to disintegrate than a rational comment. You need to be weary of leaving one-sentence explanations because there are chances they can be countered with one-sentence arguments.




I respect that you've taken the time to voice your disagreement with our choices, but your reasons have not been explained well enough, and they could be better structured. I also respect that powerful weapons are powerful, but we, the modpack team, want to also give the new players a structured experience that doesn't necessarily turn into or require a near-creative mode gameplay experience.

**Comment by IppSwitch** [comment]
Update to MrCrayFish Furnature mod - Printer

Printer Requirements - 2 blocks of iron, 1 piece of redstone, 1 piece of paper, 5 pieces of stone
Ink requirements for the Printer - 8 stone 1 ink sac = 2 ink cartridges. 
Copied Enchant book - Chose enchant book, Add to the printer. 2 ink Cartridges will produce one duplicate.

I tested a lvl 1 (Enchant I) and a lvl 5 (Enchant V)
I tested both Minecraft and Modded books. (Up to lvl 6 (VI)). 
All tested levels only required 2 ink cartridges. I believe all enchanted books will be the same cost.

This is NOT a difficult cost for what you receive. It completely removes the need for levels aside from those for the initial Enchanted book. 

Request Tweak or Removal.
